### PR TITLE
Pn 4698 fe interrogare lendpoint di user attributes per conoscere la versione informativa privacy

### PR DESCRIPTION
**Pull request by NicolaGiornetta**
## Short description
Added Privacy acceptance and approval to PA, PF and PG.

## List of changes proposed in this pull request
- Added new API methods about privacy acceptance and approval
- Extended reducers and actions in auth
- Extended existing tests

## How to test
Find out user with TOS and/or DATAPRIVACY to be accepted in any webapp (PA, PF and PG).
At the time of writing most users don't have DATAPRIVACY accepted yet so you'll easily find the ToS page when login.

### CODE REVIEW

File: packages/pn-pa-webapp/src/pages/ToSAcceptance.page.tsx [hunk 1]
```diff
@@ -46,18 +44,25 @@ const TermsOfService = ({ isFirstAccept, consentVersion }: TermsOfServiceProps)
   );
 
   const handleAccept = () => {
-    void dispatch(acceptToS(consentVersion))
+    void dispatch(acceptToS(tosConsent.consentVersion))
```
Comment (AndreaCimini90) [inline packages/pn-pa-webapp/src/pages/ToSAcceptance.page.tsx:47]: I should use the async/await logic.
I have a doubt, if tos or privacy are already accepted we can avoid to redo both api calls?

File: packages/pn-personafisica-webapp/src/pages/ToSAcceptance.page.tsx [hunk 2]
```diff
@@ -48,21 +47,28 @@ const TermsOfService = ({ isFirstAccept, consentVersion }: TermsOfServiceProps)
   );
 
   const handleAccept = () => {
-    void dispatch(acceptToS(consentVersion))
+    void dispatch(acceptToS(tosConsent.consentVersion))
```
Comment (AndreaCimini90) [inline packages/pn-personafisica-webapp/src/pages/ToSAcceptance.page.tsx:50]: Same as before

File: packages/pn-personagiuridica-webapp/src/pages/ToSAcceptance.page.tsx [hunk 3]
```diff
@@ -48,21 +46,28 @@ const TermsOfService = ({ isFirstAccept, consentVersion }: TermsOfServiceProps)
   );
 
   const handleAccept = () => {
-    void dispatch(acceptToS(consentVersion))
+    void dispatch(acceptToS(tosConsent.consentVersion))
```
Comment (AndreaCimini90) [inline packages/pn-personagiuridica-webapp/src/pages/ToSAcceptance.page.tsx:49]: Same as before

**Review by AndreaCimini90 — CHANGES_REQUESTED** [review]

File: packages/pn-pa-webapp/src/pages/ToSAcceptance.page.tsx (hunk 1 above)
Comment (NicolaGiornetta) [reply packages/pn-pa-webapp/src/pages/ToSAcceptance.page.tsx]: Done.

File: packages/pn-personafisica-webapp/src/pages/ToSAcceptance.page.tsx (hunk 2 above)
Comment (NicolaGiornetta) [reply packages/pn-personafisica-webapp/src/pages/ToSAcceptance.page.tsx]: Done.

File: packages/pn-personagiuridica-webapp/src/pages/ToSAcceptance.page.tsx (hunk 3 above)
Comment (NicolaGiornetta) [reply packages/pn-personagiuridica-webapp/src/pages/ToSAcceptance.page.tsx]: Done.

**Review by AndreaCimini90 — APPROVED** [review]

**Pull request merged**